packages/cli/src/commands/help.ts

```diff
@@ -4,11 +4,15 @@ import { ParentCommand } from './../parent.command'
 export class HelpCommand extends AbstractCommand {
   commands: (AbstractCommand | ParentCommand)[] = []
 
+  sortCommands(a: ParentCommand | AbstractCommand): number {
+    return a instanceof ParentCommand ? 1 : -1
+  }
```

Comment:
eslint hated the shorthand sort func `sort((a, b) => a instanceof ParentCommand ? 1 : -1)` don't know why, something about the ternary return 🤷